AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1401,6 +1401,7 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
 !
 windows101tricks.com###bs-newsletter-mailchimp-3
 cointelegraph.com##.subscription-form
+digitalstudiome.com#?#.panel-flexible > div.panel-flexible-inside > div.panels-flexible-row:has(> div.inside > div.panels-flexible-region > div.inside > div.panel-pane > div.pane-content > div.newsletter-wrapper)
```

Comment:
no need to make in monster selector on this site. It seems it can be hidden using this class `.sup-blk` + something else